<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Presenter</class>
 <widget class="QWidget" name="Presenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Presenter</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>11</y>
     <width>361</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Question ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_entry">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_entry">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Points:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answer_entry">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="points_entry">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>232</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
